modules/nf-core/plink/gwas/meta.yml

```diff
@@ -0,0 +1,79 @@
+name: "plink_gwas"
+description: Generate GWAS association studies
+keywords:
+  - association
+  - GWAS
+  - case/control
+tools:
+  - "plink":
+      description: "Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner."
+      homepage: "https://www.cog-genomics.org/plink"
+      documentation: "https://www.cog-genomics.org/plink/1.9/data#recode"
+      tool_dev_url: "https://www.cog-genomics.org/plink/1.9/dev"
+      licence: ["GPL"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
+  - meta4:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
```

Comment:
to facilitate reading I would indicate which metamap name is associated to which file